specfile/changelog.py

```diff
@@ -361,3 +366,49 @@ def get_raw_section_data(self) -> List[str]:
             result.extend(entry.content)
             result.extend(entry._following_lines)
         return result
+
+
+def _getent_name() -> str:
```

Comment:
Cold you decorate both of these with `@staticmethod`?

Reply:
Oh, sorry, those are not methods :sweat_smile:
Wouldn't it be better if they were?